handlers/create_lease_test.go

```diff
@@ -22,7 +45,11 @@ func TestCreateLeaseInvalidReq(t *testing.T) {
 }
 
 func TestCreateLeaseValidResp(t *testing.T) {
-	hdl := CreateLease(nil, nil)
+	t.Skip("FIXME")
```

Comment:
this is tracked in https://github.com/deis/k8s-claimer/issues/19, and may not get done in this PR